drivers/spi/spi_ll_stm32.c

```diff
@@ -318,6 +312,12 @@ static int spi_stm32_configure(struct device *dev,
 		}
 	}
 
+	if (config->operation & SPI_OP_MODE_SLAVE) {
```

Comment:
If MP1 specific, are we sure this is safe for other series?
If yes, please state it in commit message.

Reply:
stm32mp1x (and stm32h7) LL function SPI_Init seems to also define first the NSS then the mode, unlike other STM32 boards where this is not specified.
Changing the order shouldn't have bad repercussions on other boards, Zephyr SPI driver test has been passed successfully on disco_l475_iot1 board